.github/workflows/publish-helm-chart.yml

```diff
@@ -74,6 +80,66 @@ jobs:
           - name: Helm Chart Linter
             run: |
               helm lint ${{matrix.chartpath}}
+            
+          - name: SEMVER Check
+            #Checks Chart Version Number against SEMVER regex
+            run: | 
+              version=$(helm show chart ${{matrix.chartpath}} | 
+              awk '/version:/ {print $2}')
+
+              semver_regex="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\
+              (-(0|[1-9A-Za-z-][0-9A-Za-z-]*)(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]\
+              +(\.[0-9A-Za-z-]+)*)?$"
```

Comment:
This looks like hieroglyphics. A comment with a link here would be illuminating.